code/_onclick/hud/parallax.dm

```diff
@@ -248,18 +248,23 @@
 /obj/screen/parallax_layer/proc/update_o(view)
 	if(!view)
 		view = world.view
-	var/list/new_overlays = list()
-	var/count = CEILING(view/(480/world.icon_size), 1)+1
-	for(var/x in -count to count)
-		for(var/y in -count to count)
+
+	var/static/parallax_scaler = world.icon_size / 480
```

Comment:
whats the magic number `480` for, do you think it could be turned into a define or am I just missing out on what it means?

Reply:
The layers are 480x480